string name
string order
string personid
string[] features
